tortoise/models.py

```diff
@@ -192,6 +192,30 @@ def __new__(mcs, name: str, bases, attrs: dict, *args, **kwargs):
         meta_class = attrs.get("Meta", type("Meta", (), {}))
         pk_attr = "id"
 
+        # Searching for Field attributes in the class hierarchie
+        def __search_for_field_attributes(base, attrs: dict):
+            """
+            Searching for class attributes of type fields.Field
+            in the given class.
+
+            If an attribute of the class is an instance of fields.Field,
+            then it will be added to the attrs dict. But only, if the
+            key is not already in the dict. So derived classes have a higher
+            precedence. Multiple Inheritence is supported from left to right.
+
+            After checking the given class, the funkction will look into
+            the classes according to the mro.
```

Comment:
Please define `mro`

Reply:
Do you have any project wide standard regarding linking to other resources? mro is a standard feature of python and I think it should explained in depth, but I agree, that for a better understanding, I should add a short explanation.

Reply:
No, I think whatever you come up with is fine. maybe use ALL-CAPS to indicate it is an acronym at minimum?